docs/design/bundle-sources.md

```diff
@@ -0,0 +1,273 @@
+* Docker or Podman
+* Push access to a container registry, such as [Quay](https://quay.io)
+
+<h3 id="proc-building-plain-bundle-from-image">
+Procedure
+</h3>
+
+1. Verify that your Kubernetes manifests are in a flat directory at the root of your project similar to the following example:
+
+    ```sh
+    tree manifests
+    manifests
+    ├── namespace.yaml
+    ├── service_account.yaml
+    ├── cluster_role.yaml
+    ├── cluster_role_binding.yaml
+    └── deployment.yaml
+    ```
+
+    * If you are using [kustomize](https://kustomize.io) to build your manifests from templates, you must redirect the output into a single file under the `manifests/` directory by running the following command:
+
+        ```sh
+        kustomize build templates > manifests/manifests.yaml
+        ```
+
+   For more information, see [Building a plain bundle > Prerequisites](https://github.com/operator-framework/rukpak/blob/main/docs/bundles/plain.md#prerequisites).
+
+1. Create a Dockerfile at the root of your project:
+
+    ```sh
+    touch plainbundle.Dockerfile
+    ```
+
+1. Make the following changes to your Dockerfile:
+
+    *Example Dockerfile*
+
+    ```sh
+        FROM scratch
+        ADD catalog /configs
+    ```
+
+    > **Note:** Use the `FROM scratch` directive to make the size of the image smaller.
+
+1. Build an OCI-compliant image using your preferred build tool, similar to the following example. You must use an image tag that references a repository where you have push access privileges.
+
+    *Example build command*
+
+    ```sh
+    docker build -f plainbundle.Dockerfile -t \
+    quay.io/<organization_name>/<repository_name>:<image_tag> .
+    ```
+
+1. Push the image to your remote registry:
+
+    ```sh
+    docker push quay.io/<organization_name>/<repository_name>:<image_tag>
+    ```
+
+### Additional resources
+
+* [File-based catalog bundle schema](https://github.com/operator-framework/olm-docs/blob/master/content/en/docs/Reference/file-based-catalogs.md)
+* [OCI image specification](https://github.com/opencontainers/image-spec#oci-image-format-specification)
+* [RukPak > Building a plain bundle > Image source](https://github.com/operator-framework/rukpak/blob/main/docs/bundles/plain.md#image-source)
+* [RukPak > Sources > Images > Private image registries](https://github.com/operator-framework/rukpak/blob/main/docs/sources/image.md#private-image-registries)
+
+## Creating a file-based catalog
+
+If you do not have a file-based catalog, you must perform the following steps to initialize the catalog.
+
+<h3 id="prereqs-creating-a-fbc">
+Prerequisites
+</h3>
+
+* `opm` CLI tool
+* Docker or Podman
+
+<h3 id="proc-creating-a-fbc">
+Procedure
+</h3>
+
+1. Create a directory for the catalog by running the following command:
+
+    ```sh
+    mkdir <catalog_dir>
+    ```
+
+1. Create a Dockerfile that can build a catalog image by running the `opm generate dockerfile` command:
+
+    ```sh
+    opm generate dockerfile <catalog_dir>
```

Comment:
This command generates a dockerfile that has a bunch of extraneous stuff in it. I think maybe you mixed up my comments from my last review, but the Dockerfile in these steps should look like this:

```Dockerfile
FROM scratch
ADD <catalog_dir> /configs
```

Reply:
+1

Reply:
Ah yes :sweat_smile:  I mixed up your comments from the last review, sorry about that. Fixed it now in my latest commit